README.md

```diff
@@ -10,6 +10,18 @@ Register at <https://pusher.com> and use the application credentials within your
 
 You can get the Pusher PHP library via a composer package called `pusher-php-server`. See <https://packagist.org/packages/pusher/pusher-php-server>
 
+```
+$ composer require pusher/pusher-php-server
+```
+
+Or add to `composer.json` and `composer update`:
```

Comment:
We could let composer handle the version it self for the CLI based example (like we already do).